internal/connector/internal/config/connectors.go

```diff
@@ -79,22 +148,29 @@ func (c *ConnectorsConfig) ReadFiles() error {
 			// Read the file
 			buf, err := os.ReadFile(path)
 			if err != nil {
-				err = gherrors.Errorf("error reading catalog file %s: %s", path, err)
-				return err
+				return gherrors.Errorf("error reading catalog file %s: %s", path, err)
```

Comment:
You should be using `%w` verb instead of `%s` so that the underlaying error is correctly wrapped.